### PR TITLE
Fixing args and testing packages

### LINKED ISSUE #124: cli_args not defined. Had to convert cli_args in __main__.py to args
When module run `python -m stability_sdk -W 512 -H 512 "A stunning house."` , error returned `cli_args` not defined. had to change `cli_args` to `args` in __ __main____.py . and import function `process_artifacts_from_answers` from stability_sdk.client.

I'm not sure this is the right solution, so if I can receive confirmation from a developer with access saying it is, or not (with the solution) I can go ahead and close the issue.

### LINKED ISSUE #145: Bug - running inference with verbose flag to show images with PIL fails because logger is not defined
https://github.com/Stability-AI/stability-sdk/blob/67b5cf2a32fd8e8b049a3998bc33b13276130721/src/stability_sdk/utils.py#L92

### PR DESCRIPTION
**Pull request by palp**
This fixes #124, which I noticed when testing the latest release, as well as fixing the "old" (but actually working) `client.py` args to correctly use server defaults and bringing the "new" args in line with that. We should spend some effort as mentioned in #148 on cleaning up argument parsing, unifying it between old/new structures, and general testing to avoid sending out broken code - I've added a small sanity check to the release process.

Also fixes #145.

### CODE REVIEW

File: .github/workflows/publish-to-pypi.yml [hunk 1]
```diff
@@ -13,27 +13,34 @@ on:
...
-      uses: pypa/gh-action-pypi-publish@release/v1
-      with:
-        password: ${{ secrets.PYPI_API_TOKEN }}
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: "3.7"
```
Comment (wbrown) [inline .github/workflows/publish-to-pypi.yml:25]: `3.7`? Not `3.8`?

**Review by wbrown — APPROVED** [review]

File: .github/workflows/publish-to-pypi.yml (hunk 1 above)
Comment (palp) [reply .github/workflows/publish-to-pypi.yml:25]: > `3.7`? Not `3.8`?

No clue. Anything about pushing to pypi that would matter there or should we just bump it?

Comment (palp) [reply .github/workflows/publish-to-pypi.yml:25]: @dmarx any thoughts here?

Comment (dmarx) [reply .github/workflows/publish-to-pypi.yml:25]: I think when we set this up, google colab was pinned to 3.7 and I wanted to make sure the pypi package was compatible. I'm pretty sure colab uses 3.8 now, so we can probably bump the version up. will double check tomorrow.

Comment (palp) [reply .github/workflows/publish-to-pypi.yml:25]: Yup - 
image

Comment (palp) [inline .github/workflows/publish-to-pypi.yml:25]:
```suggestion
          python-version: "3.8"
```

**Review by wbrown — APPROVED** [review]

**Pull request merged**